debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
In my head it was just needed to tell that the package is in 3.0 format.
I do not rely on quilt at all.